low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -27,8 +27,15 @@ def __init__(self, args: list):
         self.name = "sspl_test"
 
     def process(self):
-        # TODO: Need to convert run_tests.sh from shell to python.
-        CMD = f"{TEST_DIR}/run_tests.sh test {' '.join(self.args)}"
+        # TODO: Need to convert run_tests.sh from shell to python
+        test_plan=None
```

Comment:
this script too can take --plan argument and ignore anything else coming.

Reply:
@cdeshmukh sspl_test.py will accept all the args from sspl_setup, but will pass only test plan arg to the run_test.sh script will ignore other args.
Do we need any other code flow?

Reply:
i understand the flow you put in place. Am saying though parsing should be generic and user friendly, which existing script is not. So adding --plan here too would help.

Its not functional part am talking about, but code maintainability & user friendliness